src/backup.ts

```diff
@@ -0,0 +1,302 @@
+      .matchClusterFromBackupName(this.name)
+      .toString();
+  }
+
+  /**
+   * This backup's identifier.
+   * @readonly
+   * @return {string}
+   */
+  get backupId(): string {
+    if (!this.name) {
+      throw new TypeError(
+        'A backup name is required to determine the backupId.'
+      );
+    }
+    return this._tableAdminClient
+      .matchBackupFromBackupName(this.name)
+      .toString();
+  }
+
+  /**
+   * A Date-compatible PreciseDate representation of `expireTime`.
+   * @readonly
+   * @return {PreciseDate}
+   */
+  get expireDate(): PreciseDate {
+    if (!this.expireTime) {
+      throw new TypeError('An expireTime is required to convert to Date.');
+    }
+    return new PreciseDate({
+      seconds: this.expireTime.seconds!,
+      nanos: this.expireTime.nanos!,
+    });
+  }
+
+  /**
+   * A Date-compatible PreciseDate representation of `startTime`.
+   * @readonly
+   * @return {PreciseDate}
+   */
+  get startDate(): PreciseDate {
+    if (!this.startTime) {
+      throw new TypeError('A startTime is required to convert to Date.');
+    }
+    return new PreciseDate({
+      seconds: this.startTime.seconds!,
+      nanos: this.startTime.nanos!,
+    });
+  }
+
+  /**
+   * A Date-compatible PreciseDate representation of `endTime`.
+   * @readonly
+   * @return {PreciseDate}
+   */
+  get endDate(): PreciseDate {
+    if (!this.endTime) {
+      throw new TypeError('An endTime is required to convert to Date.');
+    }
+    return new PreciseDate({
+      seconds: this.endTime.seconds!,
+      nanos: this.endTime.nanos!,
+    });
+  }
+
+  /**
+   * Deletes this pending or completed Cloud Bigtable backup.
+   *
+   * @param {DeleteBackupOptions} [options]
+   * @return {Promise<DeleteBackupResponse>}
+   * @see {Cluster#deleteBackup}
+   *
+   * @example <caption>include:samples/document-snippets/cluster.js</caption>
+   * region_tag:bigtable_cluster_delete_backup
+   */
+  delete(options?: DeleteBackupOptions): Promise<DeleteBackupResponse> {
+    if (!this.name) {
+      return Promise.reject(
+        new TypeError('A backup name is required to delete a backup.')
+      );
+    }
+
+    return this._bigtable
+      .instance(this.instanceId)
+      .cluster(this.clusterId)
+      .deleteBackup(this.backupId, options);
+  }
+
+  /**
+   * Gets fresh metadata for this Cloud Bigtable Backup.
+   *
+   * @param {GetBackupOptions} [options]
+   * @return {Promise<GetBackupResponse>}
+   * @see {Cluster#getBackup}
+   *
+   * @example <caption>include:samples/document-snippets/cluster.js</caption>
+   * region_tag:bigtable_cluster_get_backup
+   */
+  get(options?: GetBackupOptions): Promise<GetBackupResponse> {
+    if (!this.name) {
+      return Promise.reject(
+        new TypeError('A backup name is required to get a backup.')
+      );
+    }
+
+    return this._bigtable
+      .instance(this.instanceId)
+      .cluster(this.clusterId)
+      .getBackup(this.backupId, options);
+  }
```

Comment:
Maybe i don't understand the API's definition of "metadata," but the necessary/valuable information in the backup object (e.g. `name`, `expireTime`, `sourceTable`) has sibling properties that are more "meta" (e.g. `state`, `sizeBytes`). When you say "object," does that mean the class instance (e.g. `Table`) of the actual object (`{name: '.../tables/my-table'}`)? Otherwise, given the placement of the properties, this becomes overkill to separate the two, so for now I won't implement `getMetdata`, and _will_ implement `get`.